### ISSUE TITLE
Running set of alerts from docker - snowsec/snowalert 

### ISSUE THREAD

**Issue by bhasampa** [snowflakedb/SnowAlert#413]
Dear Team, 

Can we trigger multiple alerts as part of  single docker run? 

for example imagine I am having 20 alert rules out of which few alerts need to be executed hourly basis and remaining group of alerts need to be executed on 5 minutes frequency.

docker run --env-file snowalert-ea_bihub.envs snowsec/snowalert ./run all -- This will execute all the alerts. 

docker run --env-file snowalert-ea_bihub.envs snowsec/snowalert ./run all DQ_SONY_FILE_ALERT_QUERY -- This will execute only specific alert which we had specified.

Can you please provide me the example command which can be used to call multiple alerts? 

Thanks,
Bhaskar

**Comment by sfc-gh-afedorov** [comment]
Thanks or the request -- this is not currently possible, but I can include it the work I'm planning for the next release.

**Comment by bhasampa** [comment]
Thanks for taking up as enhancement!!

**Comment by sfc-gh-afedorov** [comment]
Sure thing, seems simple enough. The next version should support e.g. running the following command on a 2 minute cron job to get what you're looking for done, if I understood it right —

~~~
(.venv) % SA_ALERT_CUTOFF_MINUTES=3 python runners/run.py alerts AQ_80BPLTHTYX_ALERT_QUERY AQ_YUYHEEVFDF_ALERT_QUERY
[66278] AQ_80BPLTHTYX_ALERT_QUERY processing...
[66278] AQ_80BPLTHTYX_ALERT_QUERY created 2, updated 0 rows.
[66278] AQ_80BPLTHTYX_ALERT_QUERY metadata recorded.
[66278] AQ_80BPLTHTYX_ALERT_QUERY done.
[66278] RUN metadata recorded.
[66278] AQ_YUYHEEVFDF_ALERT_QUERY processing...
[66278] AQ_YUYHEEVFDF_ALERT_QUERY created 1, updated 0 rows.
[66278] AQ_YUYHEEVFDF_ALERT_QUERY metadata recorded.
[66278] AQ_YUYHEEVFDF_ALERT_QUERY done.
[66278] RUN metadata recorded.
~~~

You should be able to use this now with —

~~~
docker run --env-file snowalert-ea_bihub.envs snowsec/snowalert:1.9.5-rc ./run alerts QUERY_1_ALERT_QUERY QUERY_2_ALERT_QUERY
~~~

Let me know if you were thinking something else.

Also, a heads up — we're working on enhancements to make custom schedules possible by editing metadata in Snowflake and without having to modify the runtime piece, since the folks who want to schedule alerts often aren't the same as those who maintain the infrastructure. Feel free to shoot an email or invite if you'd like more detail of how this will work.

**Comment by bhasampa** [comment]
Dear Team, 

Yeah this is what we were expecting on the execution multiple alerts in docker. 

We are interested to hear more on metadata driven schedule alerts. We will reach out to you to get more details on it. 

Can you please share across the email id 

Thanks,
Bhaskar

**Comment by sfc-gh-afedorov** [comment]
yes, it won't be on by default, but we'll likely have an option to have alert runners run at arbitrary time intervals off of metadata in the next release

the group's email is snowalert@snowflake.com

**Comment by sfc-gh-afedorov** [comment]
fixed in 1.9.5
